brownie/network/account.py

```diff
@@ -939,13 +946,22 @@ def sign_defunct_message(self, message: str) -> SignedMessage:
         msg_hash_bytes = defunct_hash_message(text=message)
         eth_private_key = eth_keys.keys.PrivateKey(HexBytes(self.private_key))
         (v, r, s, eth_signature_bytes) = sign_message_hash(eth_private_key, msg_hash_bytes)
-        return SignedMessage(
-            messageHash=msg_hash_bytes,
-            r=r,
-            s=s,
-            v=v,
-            signature=HexBytes(eth_signature_bytes),
-        )
+        if ETH_ACCOUNT_LT_0_13_0:
+            return SignedMessage(
+                messageHash=msg_hash_bytes,
+                r=r,
+                s=s,
+                v=v,
+                signature=HexBytes(eth_signature_bytes),
+            )
+        else:
+            return SignedMessage(
```

Comment:
facepalm for upstream breaking changes lol.

is there a reason you need to support multiple versions of `eth-account`?

Reply:
yeah, it was just a dep conflict. if we're going to support multiple hexbytes we need to support multiple eth-account too because anybody pinned to hexbytes<1 can't use the current eth-account

I figured since we release eth-event in a way that serves those people, we would do the same in here